listenbrainz/webserver/static/js/src/RecentListens.tsx

```diff
@@ -391,7 +391,7 @@ export default class RecentListens extends React.Component<
         }
       });
       try {
-        const data = await this.APIService.getFeedbackForUserForRecordings(
```

Comment:
What was the issue with using this implementation? We are instantiatng the APIService again just to avoid this?

Reply:
We're not instantiating here,  it's already instantiated and available in the global context.
I'm slowly cleaning up all components to remove `this.APIService = APIService` and use the API from context directly, it's a bit cleaner and clearer.

Reply:
The reason we still have `this.APIService = APIService` in some components is just leftover from how we previously did it; I wanted to minimize the number of changes when I did the global context refactor.

Reply:
Right, makes sense!